docs/core/versions/lts-current.md

```diff
@@ -0,0 +1,36 @@
+---
+title: .NET Core Support | Microsoft Docs
+description: Learn about the different release train supports (LTS and Current) for .NET Core
+keywords: .NET, .NET Core, lts, current, fts, support, support trains, support tracks, Lifecycle, release trains
+author: kendrahavens
+ms.author: mairaw
+ms.date: 01/30/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: fedc7025-f320-4cba-957b-ef74885f66de
+---
+
+# .NET Core Support
+
+This is a general description of .NET Core support.
+
+## LTS and Current Release Trains
+
+Having two support release trains is a common concept in use throughout the software world, specially for open-source projects like .NET Core. .NET Core has two release trains: [Long Term Support (LTS)](https://en.wikipedia.org/wiki/Long-term_support) and Current. LTS releases are maintained for stability over their lifecycle, receiving fixes for important issues and security fixes. New feature work and additional bug fixes take place in Current releases. From a support perspective, these release trains have the following support lifecycle attributes.
```

Comment:
nit: I know I suggested that but now reading in the published doc I don't like it so much, but I can change that later if you'd like to get this merged soon. 
Instead of saying ".NET Core has two release trains: ", we probably should say ".NET Core has the following support release trains:". I thought that two support release trains was kind of repetitive to say twice.